lib/chainsync/chainsync/db/base/__init__.py

```diff
@@ -1,19 +1,17 @@
 """Generic database utilities"""
 
-from .conversions import convert_scaled_value_to_decimal
-from .db_schema import Base, Transaction, UserMap
-from .postgres import (
+from .interface import (
     PostgresConfig,
     TableWithBlockNumber,
-    add_transactions,
     add_user_map,
     build_postgres_config,
     close_session,
     drop_table,
     get_latest_block_number_from_table,
-    get_transactions,
     get_user_map,
     initialize_engine,
     initialize_session,
     query_tables,
 )
+from .schema import Base, UserMap
+from .utils import convert_scaled_value_to_decimal
```

Comment:
@sentilesdal has beef with "utils" because it's too generic. I don't have strong opinions here, but I think he's probably more right than not. (true in general lol) Since this is the only function in the file can we just name it as such? `from .convert_scaled_value_to_decimal import convert_scaled_value_to_decimal` is annoying, but would only need to be done within the `db/base` folder. Elsewhere it is covered by this`__init__` file.

Reply:
We've gone back and forth on this so many times, this function was intended to be a private method within convert_data.py, not to be used by anyone else. Perhaps when we find that this function is more useful generically, we can revisit this. For now, moving this function back as a private function in `convert_data.py`